package/resources/CONCLUSION.rtf

```diff
@@ -28,14 +28,20 @@ Homebrew is run entirely by unpaid volunteers. Please consider donating:\
 \pard\tx220\tx720\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\li720\fi-720\pardirnatural\partightenfactor0
 \ls3\ilvl0\cf0 \ulnone {\listtext	\uc0\u8259 	}To add Homebrew to your 
 \f2 PATH
-\f1  run 
-\f2 brew shellenv
-\f1  in your shell profile (e.g. 
+\f1 , add to your shell profile (e.g. 
 \f2 ~/.bash_profile
 \f1  or 
 \f2 ~/.zprofile
-\f1 )\
-{\listtext	\uc0\u8259 	}Run 
+\f1 ):\
+\pard\tx940\tx1440\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\li1440\fi-1440\pardirnatural\partightenfactor0
+\ls3\ilvl1\cf0 {\listtext	\uc0\u8259 	}
+\f2 eval "$(/opt/homebrew/bin/brew shellenv)"
+\f1  on Apple Silicon\
+\ls3\ilvl1{\listtext	\uc0\u8259 	}
+\f2 eval "$(/usr/local/bin/brew shellenv)"
+\f1  on Intel\
```

Comment:
Could we get away with not specifying the architecture here and just say run `eval "$(brew shellenv)"` instead like we do in the docs?

- https://docs.brew.sh/Manpage#shellenv-bashcshfishpwshshtcshzsh

Reply:
I'd compare this to the shell script installer, which [does print out the full path](https://github.com/Homebrew/install/blob/aceed88a4a062e2b41dc40a7428c71309fce14c9/install.sh#L1039).

Something actually copy-pastable seems better to me, as it doesn't seem ideal to let the user to figure out where `brew` is. I assume most of those who read the manpage already have `brew` in `PATH`.

Reply:
> I'd compare this to the shell script installer, which [does print out the full path](https://github.com/Homebrew/install/blob/aceed88a4a062e2b41dc40a7428c71309fce14c9/install.sh#L1039). (It would probably be nicer to make this text architecture-specific like the shell script installer does, but I'm not sure if that's possible.)

This would be ideal but I'm not sure how to do this either.

> Something actually copy-pastable seems better to me, as it doesn't seem ideal to let the user to figure out where `brew` is. I assume most of those who read the manpage already have `brew` in `PATH`.

Good point!